python/lsst/ctrl/bps/panda/edgenode/build_cmd_line_decoder.py

```diff
@@ -138,9 +139,8 @@ def create_idds_workflow(config_file, compute_site):
 idds_workflow = bps_workflow.idds_client_workflow
 
 _, max_copy_workers = config.search("maxCopyWorkers", opt={"default": PANDA_DEFAULT_MAX_COPY_WORKERS})
-copy_files_for_distribution(
-    bps_workflow.files_to_pre_stage, config["fileDistributionEndPoint"], max_copy_workers
-)
+file_distribution_uri = ResourcePath(config["fileDistributionEndPoint"])
```

Comment:
```suggestion
file_distribution_uri = ResourcePath(config["fileDistributionEndPoint"], forceDirectory=True)
```
This will make sure that if someone forgets the trailing `/` that it still acts like a directory on remote URIs.